x/mint/keeper/abci.go

```diff
@@ -20,76 +18,12 @@ func (k Keeper) BeginBlocker(ctx context.Context, ic types.InflationCalculationF
 		return err
 	}
 
-	params, err := k.Params.Get(ctx)
+	// we pass -1 as epoch number to indicate that this is not an epoch minting,
+	// but a regular block minting. Same with epoch id "block".
+	err = mintFn(ctx, k.Environment, &minter, "block", -1)
```

Comment:
how would you disable this?

Reply:
The mintFn always check for the epoch ID and returns early if it doesn't match. It's not the cleanest, but works :D